backend/src/auth/schemas/__init__.py

```diff
@@ -1,19 +1,22 @@
-from src.auth.schemas.endpoint_schema import (
-    LoginRequest,
-    LoginResponse,
-    RefreshTokenResponse,
-    RegisterRequest,
-    RegisterResponse,
+from src.auth.schemas.request import (
+    GetRefreshTokenModel,
+    LoginUserModel,
+    RegisterUserModel,
```

Comment:
Same as https://github.com/bh-coders/trip-planer/pull/109/files#r1510357822. Change names to be more accurate

Reply:
https://github.com/bh-coders/trip-planer/pull/109#discussion_r1510357822